src/reducers/map-style-updaters.js

```diff
@@ -189,7 +189,7 @@ function findLayerFillColor(layer) {
 function get3DBuildingColor(style) {
   // set building color to be the same as the background color.
   if (!style.style) {
-    return DEFAULT_BLDG_COLOR;
+    return hexToRgb(DEFAULT_BLDG_COLOR);
```

Comment:
hmm, maybe it's easier to save `DEFAULT_BLDG_COLOR` as a [r, g, b]?

You also need to convert it on line 206

Reply:
I initially thought about that as well but `DEFAULT_BLDG_COLOR ` at line 206 is converted to rgb at line 212; seemed like it made more sense to leave it as a hex since this function is operation on other hex values